webrtc-stats.html

```diff
@@ -88,29 +88,55 @@
         [[!WEBRTC]].
       </p>
     </section><!-- VS: getStats() remains in base spec! -->
+    <section class="informative">
+      <h2>Basic concepts</h2>
+      <p>The stats API is defined in [[!GETUSERMEDIA]]. It is defined to return
+	a set of objects that are a subclass of the RTCStats dictionary. This is
+	normatively defined in [[!GETUSERMEDIA]], but is reproduced here
+	for ease of reference.
+      </p>
+      <pre class="idl">
+	dictionary RTCStats {
+    DOMHighResTimeStamp timestamp;
+    RTCStatsType        type;
+    DOMString           id;
+	};
+      </pre>
+    </section>
     <section id="rtctatstype-*">
       <h2>
         RTCStatsType
-      </h2><code>RTCStatsType</code> object is initialized to the name of the dictionary that the
-      <code>RTCStats</code> represents.
-      <div class="note">
-        OPEN ISSUE: Need to define an IANA registry for the RTCStatsType and populate it with the
-        following set as baseline.
-      </div>
+      </h2>
+      <p>The <code>RTCStatsType</code> member indicates the type of the
+      object that the
+      <code>RTCStats</code> object represents. An object with a given "type"
+      can be only one type of dictionary, but multiple "type" values may use
```

Comment:
I think it's grammatically correct as it stands, but hard to read. Reformulating.